### PR TITLE
feat: code improvements

### PR DESCRIPTION
**Pull request by Shaquu**
- README.md linting
- `read -r` ignore backslashes as escape character
- use `-gt` or `-lt` for comparing to number
- double quote to prevent globbing and splitting
- `cd ~ || exit 1` to exit script if `cd` failed
- replace `expr` simpler code

- [x] Positive test on pi4

### CODE REVIEW

**Comment by Shaquu** [comment]
Start time: Sun 20 Feb 19:24:47 CET 2022
   End time:   Sun 20 Feb 19:43:10 CET 2022
   Duration:   1103 seconds
   Max jobs:   3
   Option:     4

File: README.md [hunk 1]
```diff
@@ -1,23 +1,24 @@
...
- - 32 and 64 bit support
- - libfdk-aac (Built for your platform)
- - libx264 (Installed via apt, or will build from source as a fallback if not available)
- - h264_omx (32Bit only, Raspberry Pi only, deprecated)
- - h264_v4l2m2m (replaces h264_omx)
- - Ability to add extra compile flags
+- FFmpeg 5.0+ (Built for your platform)
+- 32 and 64 bit support
+- libfdk-aac (Built for your platform)
+- libx264 (Installed via apt, or will build from a source as a fallback if not available)
```
Comment (marcus-j-davies) [inline README.md:11]: "Built from **a** source" or "built from source" 😉 

Its "built from source" as opposed to "built from a source"

**Review by marcus-j-davies — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Shaquu) [reply README.md:11]: You native, I am not :)
Which form do you prefer? 
I believe you can apply changes.

Comment (marcus-j-davies) [reply README.md:11]: I will add tested systems to the read me, so will make the changes here 👍

**Pull request merged**